gossip-base/src/main/java/org/apache/gossip/crdt/CrdtAddRemoveSet.java

```diff
@@ -0,0 +1,9 @@
+package org.apache.gossip.crdt;
+
+import java.util.Set;
+
+public interface CrdtAddRemoveSet<T, SetType extends Set<T>, R extends CrdtAddRemoveSet<T, SetType, R>> extends CrdtSet<T, SetType, R> {
+  R add(T element);
```

Comment:
Documentation needed here. POint out that original set is not modified and that new set is returned.